From a13f3e4baeb647cdabe4b73ac639cae16d9c42a4 Mon Sep 17 00:00:00 2001
From: "Jory A. Pratt" <anarchy@gentoo.org>
Date: Mon, 6 Apr 2020 19:14:26 +0200
Subject: [PATCH 01/37] Don't use build id

We must drop build id as it causes conflicts when merging
thunderbird/firefox/seamonkey on same system when using
splitdebug.

Signed-off-by: Thomas Deutschmann <whissi@gentoo.org>
---
 js/src/old-configure.in | 7 -------
 old-configure.in        | 7 -------
 2 files changed, 14 deletions(-)

diff --git a/js/src/old-configure.in b/js/src/old-configure.in
index 636321f598..e848bd321e 100644
--- a/js/src/old-configure.in
+++ b/js/src/old-configure.in
@@ -361,13 +361,6 @@ if test "$GNU_CC"; then
                   AC_MSG_RESULT([no])
                   LDFLAGS=$_SAVE_LDFLAGS)
 
-    AC_MSG_CHECKING([for --build-id=sha1 option to ld])
-    _SAVE_LDFLAGS=$LDFLAGS
-    LDFLAGS="$LDFLAGS -Wl,--build-id=sha1"
-    AC_TRY_LINK(,,AC_MSG_RESULT([yes]),
-                  AC_MSG_RESULT([no])
-                  LDFLAGS=$_SAVE_LDFLAGS)
-
     _DEFINES_CFLAGS="-include $jsconfdefs -DMOZILLA_CLIENT"
 fi
 
diff --git a/old-configure.in b/old-configure.in
index a05ac13cd1..4ddc3f5b45 100644
--- a/old-configure.in
+++ b/old-configure.in
@@ -349,13 +349,6 @@ if test "$GNU_CC"; then
                   AC_MSG_RESULT([no])
                   LDFLAGS=$_SAVE_LDFLAGS)
 
-    AC_MSG_CHECKING([for --build-id=sha1 option to ld])
-    _SAVE_LDFLAGS=$LDFLAGS
-    LDFLAGS="$LDFLAGS -Wl,--build-id=sha1"
-    AC_TRY_LINK(,,AC_MSG_RESULT([yes]),
-                  AC_MSG_RESULT([no])
-                  LDFLAGS=$_SAVE_LDFLAGS)
-
     AC_MSG_CHECKING([for --ignore-unresolved-symbol option to ld])
     HAVE_LINKER_SUPPORT_IGNORE_UNRESOLVED=
     _SAVE_LDFLAGS=$LDFLAGS
-- 
2.31.1

